### PR TITLE
Azure how-to-guide DNS Resolver

### PR DESCRIPTION
**Pull request by Sid-Lais**

### CODE REVIEW

**Comment by Sid-Lais** [comment]
@mishushakov please review this

File: how-to-guides/cloud-services/azure/setting-up-dns-private-resolver.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Azure DNS Private Resolver
+
+Setting up a DNS Private Resolver so that you can access the Azure Private DNS zones connected to your VNET.
```
Comment (cmiller01) [inline how-to-guides/cloud-services/azure/setting-up-dns-private-resolver.md:3]: What is VNET? Preferably spell it out first, then use the acronym later

File: .gitbook/assets/azure-dns-private-resolver-architecture.png
Comment (cmiller01) [inline .gitbook/assets/azure-dns-private-resolver-architecture.png:1]: Cool diagram! Curious what program did you use?

File: how-to-guides/cloud-services/azure/setting-up-dns-private-resolver.md (hunk 1 above)
Comment (SAM-Aryan) [reply how-to-guides/cloud-services/azure/setting-up-dns-private-resolver.md:3]: okay, will fix that right away.

File: .gitbook/assets/azure-dns-private-resolver-architecture.png
Comment (SAM-Aryan) [reply .gitbook/assets/azure-dns-private-resolver-architecture.png:1]: lucidchart

**Comment by SAM-Aryan** [comment]
@mishushakov can you please review this?

**Review by dray92 — APPROVED** [review]

**Review by dray92 — APPROVED** [review]

**Pull request merged**